internal/devbox/packages.go

```diff
@@ -80,6 +90,32 @@ func (d *Devbox) Outdated(ctx context.Context) (map[string]UpdateVersion, error)
 	return outdatedPackages, nil
 }
 
+// isGreater returns true if v1 is greater than v2
+func isGreater(v1, v2 string) bool {
```

Comment:
I wish it was this simple :(

Unfortunately, nix package versions don't follow any convention. Some (many) use semver, but many do not.

We have some pretty complicated logic to try to do this on the search server. I can look into open sourcing it so you can use it here.

Reply:
Let me know, for now this can be enough for doing this dirty job :)